lib/setup/SetupClass.php

```diff
@@ -477,7 +504,11 @@ public function importTigerData()
 
         $aDBInstances = array();
         for ($i = 0; $i < $this->iInstances; $i++) {
-            $aDBInstances[$i] =& getDB(true);
+            // https://secure.php.net/manual/en/function.pg-connect.php
+            $DSN = CONST_Database_DSN;
+            $DSN = preg_replace('/^pgsql:/', '', $DSN);
+            $DSN = preg_replace('/;/', '', $DSN);
+            $aDBInstances[$i] = pg_connect($DSN, PGSQL_CONNECT_FORCE_NEW | PGSQL_CONNECT_ASYNC);
```

Comment:
This doesn't work for me. I get:
```
2019-03-06 22:01:08 == Import Tiger data
Nominatim/build-testdb/tiger/46003.sql: ERROR: no connection to the server
Nominatim/lib/lib.php:7:
string(27) "no connection to the server"
```

Reply:
First `;` had to be replaced with space instead of empty string to make it a valid DSN. Then a `pg_ping` after connection did the trick, not sure why. Looking forward to moving the logic into tested methods in db.php